docs/develop/python/message-passing.mdx

```diff
@@ -551,23 +602,28 @@ from temporalio.common import RawValue
         ...
 ```
 
+This sample creates a `dynamic_signal` Signal.
+When an unregistered or unrecognized Signal arrives with a `(self, str, Sequence[RawValue])` signature, dynamic assignment will use this handler to manage the Signal.
+It is responsible for transforming the sequence contents into usable data in a form that the method's logic can process and act on.
+
```

Comment:
I think this is duplicating the better explanation elsewhere. The main point is the `name: str` argument. The fact that args come in in a loosely-typed `Sequence` form is a technical detail that we should try not to burden users with when they're getting their heads around the basic idea of dynamic dispatch, and trying to understant what they should be using dynamic handlers for (probably they shouldn't, we should make sure not to encourage this too much). Also the workflow and activities content here is in the wrong file since this is message-passing. None of which is your fault or mine; just noting down the things we need to fix here.

Reply:
I duplicate a lot in these sections, assuming the reader will look for only the section they want/need.

I'll de-emphasize the Sequence for all the parallel coverage.